SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -67,25 +67,34 @@ public ProjectInfoAnalysisResult Execute(AnalysisConfig config, IEnumerable<stri
 
```

Comment:
Can't you just call the main Execute(...) method referencing a project that has a sonar-project.properties file on disc?  We'll generate a new properties file but this shouldn't be in the SonarRunnerWorkingDirectory, but your new validation code should still be hit and the Execute should return result.RanToCompletion = false.